### PR TITLE
Created /sites/:siteId/members endpoint

### PR DESCRIPTION
**Pull request by jbolles94**

### CODE REVIEW

File: tests/unit/site.spec.ts [hunk 1]
```diff
@@ -3,12 +3,28 @@ import { describe, it } from 'mocha';
...
 import { ChaiRequest, TEST_DB_CLIENT } from '../config';
 import { SiteModel } from '../../src/site/model';
+import { UserModel } from '../../src/user/model';
 
 require('../config'); // recommended way of loading root hooks
 
 const URL = '/sites';
 
 describe('# SITE', () => {
+  beforeEach(async () => {
```
Comment (joshplumley) [inline tests/unit/site.spec.ts:13]: isnt this done in the config already?

File: src/utils.js [hunk 2]
```diff
@@ -151,12 +159,42 @@ async function sendMailP(mailOptions) {
...
 
 function stackToDivs(stack) {
-  return stack?.split('\n').map((line) => `<div>${line}</div>`).join('');
+  return stack
+    ?.split('\n')
+    .map((line) => `<div>${line}</div>`)
+    .join('');
+}
+
+async function checkId(res, next, model, id) {
```
Comment (joshplumley) [inline src/utils.js:168]: can you add this and then I can use the same method when you merge

```
export async function verifySiteId(
  req: express.Request,
  res: express.Response,
  next: NextFunction,
  check: 'param' | 'body' = 'body', // NEW PART
) {
  const { siteId } = check === 'body' ? req.body : req.params;
...
    }
  }
}
```

Comment (joshplumley) [reply src/utils.js:168]: Also I think some other middleware in the lot and maybe job can go away if we use this one

File: src/site/controller.ts [hunk 3]
```diff
@@ -100,7 +107,16 @@ async function getSiteMembers(_req: Request, res: Response) {
 async function assignMemberToSite(_req: Request, res: Response) {
   ExpressHandler(
     async () => {
-      res.sendStatus(501);
+      const { _id } = res.locals.data;
+      const { memberId } = _req.body;
+
+      if (!memberId) return HTTPError('Missing required arg, memberId.', 400);
```
Comment (joshplumley) [inline src/site/controller.ts:113]: Can we make a middlware check for this too? I think you can just use the one you made put pass UserModel? That way we get the check for the memberId existing in the body but not in the db

File: tests/unit/site.spec.ts (hunk 1 above)
Comment (jbolles94) [reply tests/unit/site.spec.ts:13]: The Users table is not being done there. I felt it wasn't appropriate to clear out the Users table when hitting the reset endpoint, so I only wanted to clear out the users table on these tests.

File: src/utils.js (hunk 2 above)
Comment (jbolles94) [reply src/utils.js:168]: Made an issue for this.

File: src/site/controller.ts (hunk 3 above)
Comment (jbolles94) [reply src/site/controller.ts:113]: Sure

**Pull request merged**